pkgs/development/ruby-modules/gem-config/default.nix

```
@@ -626,6 +626,10 @@ in
   ovirt-engine-sdk = attrs: {
     buildInputs = [ curl libxml2 ];
     dontBuild = false;
+    meta.badPlatforms = [
+      "aarch64-darwin"
+      "x86_64-darwin"
+    ];
```

```suggestion
    meta.broken = stdenv.isDarwin; # At least until releasing https://github.com/oVirt/ovirt-engine-sdk-ruby/pull/17
```

Is this better method?